src/Http/Http.Abstractions/src/ProblemDetails/HttpValidationProblemDetails.cs

```diff
@@ -28,6 +28,15 @@ public HttpValidationProblemDetails(IDictionary<string, string[]> errors)
     {
```

Comment:
It might be a good idea to update line 27 to be `: this((IEnumerable<KeyValuePair<string, string[]>>)errors)` to avoid duplication like you did with the static methods.

Reply:
Thank you for the suggestion. Done.